### PR TITLE
update CI and docs

### PR DESCRIPTION
**Pull request by mforets**
This PR bumps CI to run Julia v1.5 and adds some cosmetic changes to the documentation.

### CODE REVIEW

**Comment by mforets** [comment]
I think this is ready to merge, then we can tag a new version (then bump again the LazySets v1.6 build, which requires this change).

File: Project.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name = "IntervalMatrices"
 uuid = "5c1f47dc-42dd-5697-8aaa-4d102d140ba9"
-version = "0.6.2"
+version = "0.6.3"
```
Comment (schillic) [inline Project.toml:3]: Just to check: This PR did not bring any code changes, so the new version is unrelated, right?

**Review by schillic — APPROVED** [review]

File: Project.toml (hunk 1 above)
Comment (mforets) [reply Project.toml:3]: yep, this PR is only for docs changes (i added the logo and other minor changes). i'll make a new patch release ..

Comment (mforets) [reply Project.toml:3]: .. because we need [this change](https://github.com/JuliaReach/IntervalMatrices.jl/pull/159/files) to be released

**Pull request merged**